node/src/main/scala/coop/rchain/node/web/BlockInfo.scala

```diff
@@ -0,0 +1,18 @@
+//package coop.rchain.node.web
```

Comment:
Do we need this file for something?

Reply:
Oh. Sorry . My bad. It should not be included in the pr.